addon/ng2/blueprints/service/files/src/app/services/__name__/__name__.ts

```diff
@@ -6,4 +6,8 @@ export class <%= classifiedModuleName %> {
 
   constructor() {}
 
-}
\ No newline at end of file
+  doSomething() {
```

Comment:
no, we shouldn't be scaffolding stuff like this. just leave the service empty for now please